lxd/daemon.go

```diff
@@ -2103,7 +2103,7 @@ func cancelCancelableOps() error {
 		}
 
 		if opAPI.MayCancel {
-			_, _ = op.Cancel()
+			_ = op.Cancel()
```

Comment:
I think we need to also `op.Wait` here to wait for the cancelled operations to finish before we move on with the shutdown process.